docs/SyncVsAsync.md

```diff
@@ -0,0 +1,93 @@
+---
+id: sync-vs-async
+title: Sync mode vs async
+---
+
+## Async mode
+
+Every browser/element function call returns a Promise and needs to be awaited to get result.
+
+It's not allowed to chain functions.
+
+Example
+
+```js
+describe('suite async', () => {
+    it('test', async () => { // pay attention to `async` keyword
+        const el = await $('body') // note `await` keyword
+        await el.click()
+
+        await browser.pause(500)
+
+        console.log(browser.capabilities) // static properties should not be awaited
+
+        await $('body').click() // WON'T WORK! You can't chain functions like this.
+    })
+})
+```
+
+### Common issues in async mode
+
+- `await $('body').click()` throws `$(...).click is not a function` because the element was not awaited. To fix this first await element then do click, like this:
```

Comment:
```suggestion
- `await $('body').click()` throws `$(...).click is not a function` because the element was not awaited. To fix this, first await element then trigger the click, like so:
```